config/crd/bases/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -179,6 +179,12 @@ spec:
                       before enable component Does not support enabled ModelMeshServing
                       at the same time
                     properties:
+                      defaultDeploymentMode:
```

Comment:
you will need to get manifests updates as well.
only change it here will break the build.

Reply:
The original change is written in the Kserve struct. I did not directly edit the CRD. Is there anywhere else I need to define the new field other than the Kserve struct?

Reply:
no, the code is correct, but to make the build work , we will need catalogsource to update, so the midstream do not break eventually. just run "make" should be enough to update bundles

Reply:
ran make before submitting a new commit. I saw that the `opendatahub-operator/bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml` file is now updated. Hopefully that is what you were talking about?